src/HotChocolate/Data/src/Data/Filters/Naming/FilterInputTypeNameDependencyDescriptor_1.cs

```diff
@@ -0,0 +1,36 @@
+using System;
+using HotChocolate.Types;
+using HotChocolate.Types.Descriptors;
+
+namespace HotChocolate.Data.Filters
+{
+    internal class FilterInputTypeNameDependencyDescriptor<T>
+        : IFilterInputTypeNameDependencyDescriptor<T>
+    {
+        private readonly IFilterInputTypeDescriptor<T> _descriptor;
+        private readonly Func<INamedType, NameString> _createName;
+
+        public FilterInputTypeNameDependencyDescriptor(
+            IFilterInputTypeDescriptor<T> descriptor,
+            Func<INamedType, NameString> createName)
+        {
+            _descriptor = descriptor
+                ?? throw new ArgumentNullException(nameof(descriptor));
```

Comment:
_descriptor = descriptor
                ?? throw new ArgumentNullException(nameof(descriptor));

OR 

_descriptor = descriptor ?? 
     throw new ArgumentNullException(nameof(descriptor));

you are using both formattings